src/collectionProcess.ts

```diff
@@ -0,0 +1,55 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+/**
+ * This script is the entry point of the code that is run in a separate 'node' process for collection of performance counters.
+*/
+'use strict';
+import debugLogger = require('debug');
+
+import { CollectorMethodType, CollectorResponse, ConstructCountersType, isConstructCountersType } from './remoteCollector';
+
+import { Counters } from './counters';
+import { jsonDump } from './utils';
+
+const trace = debugLogger('adstest:remotecollector:trace');
+
+console.log("Process: " + process.argv[1] + " has now started.");
+let counters: Counters;

```

Comment:
You use console.log in some places but trace/debug in others? Why not stay consistent?